test/pylib/scylla_cluster.py

```diff
@@ -892,7 +892,7 @@ async def _cluster_remove_node(self, request: aiohttp.web.Request) -> aiohttp.we
         # initate remove
         try:
             await self.cluster.api.remove_node(initiator.ip_addr, to_remove.host_id, ignore_dead)
-        except RuntimeError as exc:
```

Comment:
Re commit message: please use "test/pylib: scylla_cluster:" or "test.py" as the prefix

For the PR title, I think the umbrella-prefix "test.py" is appropriate.

We're not changing anything in `pytest` :)

Reply:
@nyh said `test.py` was not a good prefix for tests themselves and not the `test.py`... framework (?).

I put `pytest` because it could mark it's modifying the "module" of pytests. Another option could be 
`pytests`. I changed it to `test/topology` for now.

Reply:
If I understand correctly, you changed the cover letter but not the title of this patch.
You can use test/topology as a prefix, but if you believe that test/pylib will be used by multiple subdirectories and not just test/toplogy, you can use the "test/pylib" prefix.

Reply:
As @nyh mentioned, the title of the commit is still `pytest: ...`

Please change to `test/pylib: ...`.